modules/ossm-supported-configurations.adoc

```diff
@@ -16,11 +16,10 @@ The following configurations are supported for the current release of {ProductNa
 Red Hat OpenShift Online is not supported for {ProductName}.
 ====
 
-* The deployment must be contained to a single {product-title} cluster that is not federated.
 * This release of {ProductName} is only available on {product-title} x86_64, IBM Z, and IBM Power Systems.
 ** IBM Z is only supported on {product-title} 4.6 and later.
 ** IBM Power Systems is only supported on {product-title} 4.6 and later.
-* Configurations where all {ProductShortName} components are contained in the {product-title} cluster in which it operates. {ProductName} does not support management of microservices that reside outside of the cluster, or in a multi-cluster scenario.
+* Configurations where all {ProductShortName} components are contained in the {product-title} cluster in which the {ProductShortName} operates. {ProductName} does not support management of microservices that reside outside of the cluster.
```

Comment:
Maybe it's simpler to say something like:
```
 * Configurations where all {ProductShortName} components are contained within a single {product-title} cluster.  {ProductName} does not support management of microservices that reside outside the cluster within which {ProductShortName} is running.
```

Reply:
@rcernich the change has been submitted.